internal/wrexec/recording_cmd.go

```diff
@@ -160,11 +169,16 @@ func (rf *RecordingCommandFactory) Wrap(ctx context.Context, logger log.Logger,
 // os/exec.Cmd, while also setting up the ShouldRecordFunc currently set in the
 // factory. It uses repoName to create a new Redis list using it.
 func (rf *RecordingCommandFactory) WrapWithRepoName(ctx context.Context, logger log.Logger, repoName api.RepoName, cmd *exec.Cmd) *RecordingCmd {
-	store := rcache.NewFIFOList(KeyPrefix+fmt.Sprintf("-%s", repoName), rf.maxSize)
+	store := rcache.NewFIFOList(GetFIFOListKey(string(repoName)), rf.maxSize)
 	return RecordingWrap(ctx, logger, rf.shouldRecord, store, cmd)
 }
 
 // NewNoOpRecordingCommandFactory is a recording command factory that is intialised with a nil shouldRecord and maxSize 0. This is a helper for use in tests.
 func NewNoOpRecordingCommandFactory() *RecordingCommandFactory {
 	return &RecordingCommandFactory{shouldRecord: nil, maxSize: 0}
 }
+
+// GetFIFOListKey returns the name of FIFO list in Redis for a given repo name.
+func GetFIFOListKey(repoName string) string {
+	return KeyPrefix + fmt.Sprintf("-%s", repoName)
```

Comment:
Since we're using the `fmt.Sprintf` function, we might as well use it to construct the full key.

```suggestion
	return fmt.Sprintf("%s-%s", KeyPrefix, repoName)
```

Reply:
yep, agree!